twilight-mention/src/fmt.rs

```diff
@@ -118,6 +166,51 @@ impl Mention<Id<ChannelMarker>> for Id<ChannelMarker> {
     }
 }
 
+/// Mention a command. This will format as `</NAME:COMMAND_ID>`.
+impl<N> Mention<CommandMention> for (N, Id<CommandMarker>)
```

Comment:
this could be a struct instead of a tuple for clarity

Reply:
removed